src/components/Accordion/Accordion.stories.tsx

```diff
@@ -12,7 +12,7 @@ export default {
     component: Accordion,
 };
 const checkboxes = [
-    <li>
+    <li key="one">
```

Comment:
Not strictly part of this PR but it bothered me seeing this warning every time Storybook opened.